html/template/default/assets/css/style.css

```
@@ -1711,7 +1704,7 @@ textarea.form-control-lg {
 .form-floating > .form-control-plaintext::placeholder {
   color: transparent;
 }
-.form-floating > .form-control:not(:-moz-placeholder-shown), .form-floating > .form-control-plaintext:not(:-moz-placeholder-shown) {
+.form-floating > .form-control:not(:-moz-placeholder), .form-floating > .form-control-plaintext:not(:-moz-placeholder) {
```

Stylelint (17.11.0)

[error] Vendor-prefixed selector ":-moz-placeholder" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)